pkg/storage/replica_test.go

```diff
@@ -9106,12 +9124,26 @@ func TestReplicaLocalRetries(t *testing.T) {
 			},
 			batchFn: func(ts hlc.Timestamp) (ba roachpb.BatchRequest, expTS hlc.Timestamp) {
 				ba.Txn = newTxn("c", ts.Prev())
-				cput := cPutArgs(roachpb.Key("c"), []byte("cput"), []byte("put"))
+				cput := cPutArgs(roachpb.Key("c"), []byte("iput"), []byte("put"))
```

Comment:
Strange that this line changed. Not worth diverging from master to fix though.

Reply:
Yes, that was just a mistake, but it doesn't affect the test whatsoever.